apps/klaviyo/pages/_app.tsx

```diff
@@ -24,14 +53,19 @@ const themeOverrides: Partial<Theme> = {
   },
 };
 
+const generateClassName = createGenerateClassName({
+  productionPrefix: "c",
```

Comment:
What `c` stands for? :thinking:

Reply:
It was copy pasted from stack overflow 😆 

Bascially its some workaround to material/jss that scopes generated styles so they don't conflict. 
Without that, code from `packages/shared` was duplicating classnames